### PR TITLE
feat(text): L3-378 update text component and text and breakpoint tokens

### PR DESCRIPTION
**Pull request by scottdickerson**
**Jira ticket**

[L3-378](https://phillipsauctions.atlassian.net/browse/L3-378)

**Screenshots**
Added new breakpoints story:
image

**Summary**

These changes are feedback from the design review of the About Us page with @RyanOlanoPHILLIPS @davidicus and @adietrich3074 

This updates token names for breakpoints, makes some changes to the Text component to better render blockquotes, makes some small tweaks to the semantic HTML that gets rendered.  

**Change List (describe the changes made to the files)**
- change(HeroBanner): use variables for breakpoints not hardcoded
- change(*.scss): update token names
- add(breakpoint-tokens): simple breakpoint story (outside of the Grid tokens story)
- change(Text): changes to semantic html and padding for the variants

**Acceptance Test (how to verify the PR)**

- Verify the new breakpoints story
- Verify that the Text variant=blockquote renders correctly

**Regression Test**

- Check all the Text variants vs the deployed seldon
- Check the components using each of the changed scss files (Footer, etc) vs the deployed seldon to make sure no changes occur

**Evidence of testing**

- image

**Things to look for during review**

- [ ] Make sure all references to `phillips` class prefix is using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] PR should link and close out an existing issue
- [ ] Make sure all commits messages follow convention and are appropriate for the changes

New Components

- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file

[L3-387]: https://phillipsauctions.atlassian.net/browse/L3-387?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[L3-378]: https://phillipsauctions.atlassian.net/browse/L3-378?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Text/types.ts [hunk 1]
```diff
@@ -16,5 +16,5 @@ export enum TextVariants {
   ctaSm = 'ctaSm',
   email = 'email',
   label = 'label',
-  textBadge = 'textBadge',
+  badge = 'badge',
```
Comment (davidicus) [inline src/components/Text/types.ts:19]: Needs to be updated in story control dropdown

File: src/design/breakpoint-tokens/breakpoint-tokens.mdx
Comment (davidicus) [inline src/design/breakpoint-tokens/breakpoint-tokens.mdx:1]: Should we add this is we want consumers to use the media mixin? Maybe we add a section showing the mixin and how those values map the breakpoints?

File: src/components/Text/types.ts (hunk 1 above)
Comment (scottdickerson) [reply src/components/Text/types.ts:19]: this looks right to me.  Sometimes you have to close and reopen storybook:
image

File: src/design/breakpoint-tokens/breakpoint-tokens.mdx
Comment (scottdickerson) [reply src/design/breakpoint-tokens/breakpoint-tokens.mdx:1]: what do you think about this? 
image

Comment (davidicus) [reply src/design/breakpoint-tokens/breakpoint-tokens.mdx:1]: Do we want to to add a part of the table that maps the `$size-x` token used in the breakpoint to the breakpoint value?

File: src/components/Text/types.ts (hunk 1 above)
Comment (davidicus) [reply src/components/Text/types.ts:19]: Looks good now

**Review by davidicus — COMMENTED** [review]
LGTM. Just the one comment about mapping the media tokens to their breakpoint

File: src/design/breakpoint-tokens/breakpoint-tokens.mdx
Comment (scottdickerson) [reply src/design/breakpoint-tokens/breakpoint-tokens.mdx:1]: hmm, maybe not yet, lets see what people think.

**Pull request merged**